<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
background-image:url(:/table/table.jpg)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>770</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="instrBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>771</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>INSTRUCTION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_instruction">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_player">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_crupier">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>181</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="standBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>101</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>stand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hitBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>81</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>hit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doubleDownBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Double Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insuranceBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Insurance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_player1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_croupier1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_points_player">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_points_croupier">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_money">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="money1000Btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#money1000Btn {
background-color: transparent;
background-image:url(:/chips/1000.png);
border: none;
background-repeat: none;
}
#money1000Btn:hover
{
	background-image:url(:/chips/1000hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="money10Btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#money10Btn {
background-color: transparent;
background-image:url(:/chips/10.png);
border: none;
background-repeat: none;	
}
#money10Btn:hover
{
	background-image:url(:/chips/10hover.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="money20Btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#money20Btn {
background-color: transparent;
background-image:url(:/chips/20.png);
border: none;
background-repeat: none;
}
#money20Btn:hover
{
	background-image:url(:/chips/20hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="money50Btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#money50Btn {
background-color: transparent;
background-image:url(:/chips/50.png);
border: none;
background-repeat: none;
}
#money50Btn:hover
{
	background-image:url(:/chips/50hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="money100Btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>380</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#money100Btn {
background-color: transparent;
background-image:url(:/chips/100.png);
border: none;
background-repeat: none;
}
#money100Btn:hover
{
	background-image:url(:/chips/100hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="money200Btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#money200Btn {
background-color: transparent;
background-image:url(:/chips/200.png);
border: none;
background-repeat: none;
}
#money200Btn:hover
{
	background-image:url(:/chips/200hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="money300Btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#money300Btn {
background-color: transparent;
background-image:url(:/chips/300.png);
border: none;
background-repeat: none;
}
#money300Btn:hover
{
	background-image:url(:/chips/300hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="money500Btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#money500Btn {
background-color: transparent;
background-image:url(:/chips/500.png);
border: none;
background-repeat: none;
}
#money500Btn:hover
{
	background-image:url(:/chips/500hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>770</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>DOBIERZ KARTY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currently_money">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>230</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_player2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_player3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_player4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_player5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_player6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_croupier2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_croupier3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_croupier4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_croupier5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_croupier6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="zakoncz_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>771</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>230</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="betBtn">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>290</y>
      <width>221</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>bet</string>
    </property>
   </widget>
   <zorder>label_result</zorder>
   <zorder>label_money</zorder>
   <zorder>zakoncz_Btn</zorder>
   <zorder>label_currently_money</zorder>
   <zorder>instrBtn</zorder>
   <zorder>label_name_player</zorder>
   <zorder>label_name_crupier</zorder>
   <zorder>standBtn</zorder>
   <zorder>label_points_player</zorder>
   <zorder>label_points_croupier</zorder>
   <zorder>startBtn</zorder>
   <zorder>hitBtn</zorder>
   <zorder>label_instruction</zorder>
   <zorder>doubleDownBtn</zorder>
   <zorder>insuranceBtn</zorder>
   <zorder>label_card_player1</zorder>
   <zorder>label_card_player2</zorder>
   <zorder>label_card_player3</zorder>
   <zorder>label_card_player4</zorder>
   <zorder>label_card_player5</zorder>
   <zorder>label_card_player6</zorder>
   <zorder>label_card_croupier1</zorder>
   <zorder>label_card_croupier2</zorder>
   <zorder>label_card_croupier3</zorder>
   <zorder>label_card_croupier4</zorder>
   <zorder>label_card_croupier5</zorder>
   <zorder>label_card_croupier6</zorder>
   <zorder>label_bet</zorder>
   <zorder>betBtn</zorder>
   <zorder>playBtn</zorder>
   <zorder>money10Btn</zorder>
   <zorder>money20Btn</zorder>
   <zorder>money50Btn</zorder>
   <zorder>money100Btn</zorder>
   <zorder>money1000Btn</zorder>
   <zorder>money500Btn</zorder>
   <zorder>money300Btn</zorder>
   <zorder>money200Btn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
